src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2198,34 +2206,40 @@ because xbuild doesn't support framework reference assemblies.
-  Outputs="$(IntermediateOutputPath)android\AndroidManifest.xml;$(_AcwMapFile);$(_AndroidTypeMappingJavaToManaged);$(_AndroidTypeMappingManagedToJava)">
+  Outputs="$(IntermediateOutputPath)_javastubs.stamp">
   <GenerateJavaStubs
     ResolvedAssemblies="@(_ResolvedAssemblies)"
     ResolvedUserAssemblies="@(_ResolvedUserAssemblies)"
     ErrorOnCustomJavaObject="$(AndroidErrorOnCustomJavaObject)"
-	ManifestTemplate="$(_AndroidManifestAbs)"
-	MergedManifestDocuments="@(ExtractedManifestDocuments)"
-	Debug="$(AndroidIncludeDebugSymbols)"
-	NeedsInternet="$(AndroidNeedsInternetPermission)"
-	AndroidSdkPlatform="$(_AndroidApiLevel)"
-	AndroidSdkDir="$(_AndroidSdkDirectory)"
-	PackageName="$(_AndroidPackage)"
-	ManifestPlaceholders="$(AndroidManifestPlaceholders)"
-	OutputDirectory="$(IntermediateOutputPath)android"
-	MergedAndroidManifestOutput="$(IntermediateOutputPath)android\AndroidManifest.xml"
+    ManifestTemplate="$(_AndroidManifestAbs)"
+    MergedManifestDocuments="@(ExtractedManifestDocuments)"
+    Debug="$(AndroidIncludeDebugSymbols)"
+    NeedsInternet="$(AndroidNeedsInternetPermission)"
+    AndroidSdkPlatform="$(_AndroidApiLevel)"
+    AndroidSdkDir="$(_AndroidSdkDirectory)"
+    PackageName="$(_AndroidPackage)"
+    ManifestPlaceholders="$(AndroidManifestPlaceholders)"
+    OutputDirectory="$(IntermediateOutputPath)android"
+    MergedAndroidManifestOutput="$(IntermediateOutputPath)android\AndroidManifest.xml"
     UseSharedRuntime="$(AndroidUseSharedRuntime)"
-	EmbedAssemblies="$(EmbedAssembliesIntoApk)"
-	ResourceDirectory="$(MonoAndroidResDirIntermediate)"
-	BundledWearApplicationName="$(BundledWearApplicationPackageName)"
-	PackageNamingPolicy="$(AndroidPackageNamingPolicy)"
-	ApplicationJavaClass="$(AndroidApplicationJavaClass)"
-	FrameworkDirectories="$(_XATargetFrameworkDirectories);$(_XATargetFrameworkDirectories)Facades"
-	AcwMapFile="$(_AcwMapFile)">
-  </GenerateJavaStubs>
+    EmbedAssemblies="$(EmbedAssembliesIntoApk)"
+    ResourceDirectory="$(MonoAndroidResDirIntermediate)"
+    BundledWearApplicationName="$(BundledWearApplicationPackageName)"
+    PackageNamingPolicy="$(AndroidPackageNamingPolicy)"
+    ApplicationJavaClass="$(AndroidApplicationJavaClass)"
+    FrameworkDirectories="$(_XATargetFrameworkDirectories);$(_XATargetFrameworkDirectories)Facades"
+    AcwMapFile="$(_AcwMapFile)"
+  />
   <ConvertResourcesCases 
-	ResourceDirectories="$(MonoAndroidResDirIntermediate);@(LibraryResourceDirectories)"
-	ResourceNameCaseMap="$(_AndroidResourceNameCaseMap)"
-	AcwMapFile="$(_AcwMapFile)" />
+    ResourceDirectories="$(MonoAndroidResDirIntermediate);@(LibraryResourceDirectories)"
+    ResourceNameCaseMap="$(_AndroidResourceNameCaseMap)"
+    AcwMapFile="$(_AcwMapFile)"
+    AndroidConversionFlagFile="$(IntermediateOutputPath)_javastubs.stamp"
+  />
+  <Touch Files="$(IntermediateOutputPath)_javastubs.stamp;$(_AndroidResgenFlagFile)" AlwaysCreate="True" />
```

Comment:
This is the part I was worried about, will touching `$(_AndroidResgenFlagFile)` have any ill effects?

This is what made this PR green, and prevented the two `ConvertResourcesCases` from fighting each other.